Essentials/src/main/java/com/earth2me/essentials/commands/Commandseen.java

```diff
@@ -102,6 +103,9 @@ private void seenOnline(final CommandSource sender, final User user, final boole
 
         user.setDisplayNick();
         sender.sendMessage(tl("seenOnline", user.getDisplayName(), DateUtil.formatDateDiff(user.getLastLogin())));
+        if (!sender.isPlayer() || ess.getUser(sender.getPlayer()).isAuthorized("essentials.seen.joined")) {
+            sender.sendMessage(tl("whoisJoined", DateUtil.formatDateDiff(user.getBase().getFirstPlayed()), new SimpleDateFormat("yyyy/MM/dd HH:mm").format(user.getBase().getFirstPlayed())));
```

Comment:
the `SimpleDateFormat` should be moved to a field instead of create a new instance every time.